lib/modules/datasource/helm/schema.ts

```diff
@@ -0,0 +1,68 @@
+import { z } from 'zod';
+import { LooseRecord, Yaml } from '../../../util/schema-utils';
+import type { Release } from '../types';
+import { githubRelease, isPossibleChartRepo } from './common';
+
+const HelmReleaseSchema = z.object({
+  version: z.string(),
+  created: z.string().nullable().catch(null),
+  digest: z.string().optional().catch(undefined),
+  home: z.string().optional(),
+  sources: z.array(z.string()).optional(),
+  urls: z.array(z.string()),
```

Comment:
```suggestion
  urls: z.array(z.string()).optional (),
```
it's not a mandatory helm field